CHANGES.rst

```diff
@@ -527,6 +527,9 @@ astropy.io.fits
 - Fix creating ``CompImageHDU`` from header with BSCALE/BZERO: keywords are now
   ignored, as done in ``ImageHDU``. [#17237]
 
+- Fix a bug in ``nddata.Cutout2D`` when creating partial cutouts of ``Section`` objects by
+  adding a ``dtype`` property to the ``Section`` class. [#17611]
+
```

Comment:
This file must not be edited, it will be updated during the release process. You already adding a changelog fragment in `docs/changes/` so you can just revert the change here.